CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Unreleased
 
 * Drop support for Ruby 3.0. The minimum required Ruby version is now 3.1
+* No longer require a GA custom dimension to be passed into the GovukAbTesting::AbTest object
```

Comment:
Should we flag this as BREAKING? As dimension is part of the public API any code that initialises this class with dimension as an argument will now error.

Reply:
Yes I think it makes sense to tag it as breaking and change the major version.